db/migrate/20191015080931_add_temp_location_for_media_file.rb

```diff
@@ -0,0 +1,5 @@
+class AddTempLocationForMediaFile < ActiveRecord::Migration
+  def change
+  	add_column :medias, :file_tmp, :string
```

Comment:
What is this column used for?

Reply:
no need to this column as I used `process_in_background` option